lighthouse-core/report/v2/report-renderer.js

```diff
@@ -25,19 +57,183 @@
  */
 window.ReportRenderer = class ReportRenderer {
   /**
-   * @param {!Object} report
+   * @param {Document} document
+   */
+  constructor(document) {
+    this._document = document;
+  }
+
+  /**
+   * @param {ReportJSON} report
+   * @return {Element}
+   */
+  renderReport(report) {
+    try {
+      return this._renderReport(report);
+    } catch (e) {
+      return this._renderException(e);
+    }
+  }
+
+  /**
+   * @param {string} name
+   * @param {string=} className
+   * @param {Object=} attrs Attribute key/val pairs.
+   * @returns {Element}
+   */
+  _createElement(name, className, attrs = {}) {
+    const element = this._document.createElement(name);
+    if (className) {
+      element.className = className;
+    }
+    for (const [key, val] of Object.entries(attrs)) {
+      element.setAttribute(key, val);
+    }
+    return element;
+  }
+
+  /**
+   * @param {number} score
+   * @param {string} title
+   * @param {string} description
+   * @return {Element}
+   */
+  _renderScore(score, title, description) {
+    const element = this._createElement('div', 'lighthouse-score');
+
+    const value = element.appendChild(this._createElement('div', 'lighthouse-score__value'));
+    value.textContent = formatNumber(score);
+    value.classList.add(`lighthouse-score__value--${calculateRating(score)}`);
+
+    const column = element.appendChild(this._createElement('div', 'lighthouse-score__text'));
+    column.appendChild(this._createElement('div', 'lighthouse-score__title')).textContent = title;
+    column.appendChild(
+      this._createElement('div', 'lighthouse-score__description')).textContent = description;
+
+    return element;
+  }
+
+  /**
+   * @param {DetailsJSON} details
+   * @return {Element}
    */
-  constructor(report) {
-    this._report = report;
+  _renderDetails(details) {
+    switch (details.type) {
+    case 'text':
```

Comment:
looks like it's because the `/* eslint indent: ["error", 2, { "outerIIFEBody": 0 }] */` line in the file is [overriding our default](https://github.com/GoogleChrome/lighthouse/blob/bf6a0a4ca8875df3ccfb973f71e6a1eae79451d4/.eslintrc.js#L11-L14), which calls out the switch thing explicitly. /* eslint indent: [2, 2, { "SwitchCase": 1, "outerIIFEBody": 0 }] */ should work in that case